Dockerfile

```diff
@@ -22,15 +22,20 @@ COPY src/package.json ${GITLAB_CE_PAGES_WEBHOOK_DIR}/
 RUN npm install
 RUN npm install -g nodemon
 
+ENV TINI_VERSION v0.10.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+RUN chmod +x /tini
+ENTRYPOINT ["/tini", "--"]
+
 COPY entrypoint.sh /
 COPY config/nginx.conf /etc/nginx/nginx.conf
 RUN rm -f /etc/nginx/conf.d/*
+RUN chown -R nginx: /var/log/nginx
```

Comment:
Hm, then i guess i was mislead by the file or directory error not found. In this case, if output still is being shown, everything's alright.